pkg/machines/libvirt.es6

```diff
@@ -170,29 +174,47 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    INITIALIZE () {
```

Comment:
Why not reuse the `init` method above?
Please note, the `init()` can return both `boolean` or `Promise`.

I think we can fire reading of libvirt service name in `init()`, store result into redux and reuse it in GET_ALL_VMS

Reply:
Please align with similar initialization flow in `pkg/ovirt`

Reply:
We decided it is better not to mix init() in libvirt and ovirt